src/components/List/List.stories.tsx

```diff
@@ -38,12 +39,24 @@ export default {
 } as ComponentMeta<typeof List>;
 
 const Template: ComponentStory<typeof List> = () => (
-  <div className={classes['container']}>
-    <List>
-      <ListItem>List Item 1</ListItem>
-      <ListItem>List Item 2</ListItem>
-      <ListItem>List Item 3</ListItem>
-    </List>
+  <div>
+    <div className={classes['container']}>
+      <h3>Liste med dashed border</h3>
+      <List borderStyle={BorderStyle.Dashed}>
+        <ListItem>List Item 1</ListItem>
+        <ListItem>List Item 2</ListItem>
+        <ListItem>List Item 3</ListItem>
+      </List>
+    </div>
+    <br></br>
+    <div className={classes['container']}>
+      <h3>Liste med solid border</h3>
+      <List borderStyle={BorderStyle.Solid}>
+        <ListItem>List Item 1</ListItem>
+        <ListItem>List Item 2</ListItem>
+        <ListItem>List Item 3</ListItem>
+      </List>
+    </div>
   </div>
 );
```

Comment:
I think this perhaps should be reverted, and instead create separate examples for each of the variants. That way we will get automatic controls that we can toggle in Storybook UI to see how the component changes. 

Can see [how it is done in `Button` component](https://github.com/Altinn/altinn-design-system/blob/main/src/components/Button/Button.stories.tsx#L44-L78) as an example 

In the future we will also add some documentation for when to use which of the variants, which is why we want to separate the examples a bit.